src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -156,3 +156,16 @@ def mass_transfer_term(b, t, s, k, l, m):
             self.mscontactor.stream_component_interactions,
             rule=mass_transfer_term,
         )
+
+        self.clean_sx_pe_tank_cap = Var(initialize = 1405, units=units.gal, doc="Volume of polyethylene tanks")
+        self.num_sx_pe_tank = Var(initialize = 5, doc="Number of polyethylene tanks")
+
+        self.clean_sx_tank_mixer_pow = Var(initialize = 1, units=units.hp, doc="Power of tank mixers")
+        self.num_sx_tank_mixer = Var(initialize = 2, doc="Number of mixers")
+
+        self.clean_sx_process_pump_feed = Var(initialize = 281, units=units.gal/units.min, doc="Process pump feed rates")
+        self.num_sx_process_pump = Var(initialize = 3, doc="Number of process pumps")
+
+        self.clean_sx_mix_set_cap = Var(initialize = 2444, units=units.gal, doc="Volume of mixer settler tanks")
+        self.num_sx_mix_set = Var(initialize = 6, doc="Number of mixer settler tanks")
```

Comment:
Whilst these variables are nice, there is no point in adding them without having the associated correlations; otherwise these are just free variables with no defined value. Also, things like the pump generally should not be part of the unit model - there instead should be a separate pump in the flowsheet to capture this.

Reply:
Okay, actually Marcus asked me to type these in the model, so that these variables need not be hardcoded every time and can be called from the model